<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LookFeelToolbars</class>
 <widget class="QWidget" name="LookFeelToolbars">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LookFeelToolbarsUI</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Toolbar:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_toolbars">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="IconButton" name="pb_addToolbar">
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="IconButton" name="pb_deleteToolbar">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
       <property name="shortcut">
        <string>Alt+D</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line2">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="w_toolbarName" native="true">
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="textLabel20">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_toolbarName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="ck_toolbarOn">
       <property name="toolTip">
        <string>Specifies, whether toolbar is enabled and visible.</string>
       </property>
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_toolbarLocked">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Locked</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line3">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="3" column="3">
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>25</width>
         <height>101</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3">
      <widget class="IconToolButton" name="tb_up">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="psiIconName" stdset="0">
        <string>psi/arrowUp</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="3">
      <widget class="QListWidget" name="lw_selectedActions">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="IconToolButton" name="tb_down">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="psiIconName" stdset="0">
        <string>psi/arrowDown</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="IconToolButton" name="tb_right">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="psiIconName" stdset="0">
        <string>psi/arrowRight</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>25</width>
         <height>91</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel25">
       <property name="text">
        <string>A&amp;vailable actions:</string>
       </property>
       <property name="buddy">
        <cstring>tw_availActions</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="IconToolButton" name="tb_left">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="psiIconName" stdset="0">
        <string>psi/arrowLeft</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="3">
      <widget class="QTreeWidget" name="tw_availActions">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="textLabel24">
       <property name="text">
        <string>Curr&amp;ent actions:</string>
       </property>
       <property name="buddy">
        <cstring>lw_selectedActions</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IconToolButton</class>
   <extends>QToolButton</extends>
   <header>icontoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>IconButton</class>
   <extends>QPushButton</extends>
   <header>iconbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cb_toolbars</tabstop>
  <tabstop>pb_addToolbar</tabstop>
  <tabstop>pb_deleteToolbar</tabstop>
  <tabstop>le_toolbarName</tabstop>
  <tabstop>ck_toolbarOn</tabstop>
  <tabstop>ck_toolbarLocked</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
